src/plugin-slots/DesktopUserMenuToggleSlot/README.md

```diff
@@ -0,0 +1,77 @@
+# Learning User Menu Toggle Slot
+
+### Slot ID: `org.openedx.frontend.layout.header_desktop_user_menu_toggle.v1`
+
+### Slot ID Aliases
+* `desktop_user_menu_toggle_slot`
+
+## Description
+
+This slot is used to replace/modify/hide the contents of the user menu toggle button on desktop sized screens.
+
+## Examples
+
+### Modify Username Text
+
+The following `env.config.jsx` will modify the 'username' text to something more generic:
+
+![Screenshot of modified label](./images/desktop_user_menu_modified_toggle.png)
+
+```jsx
+import { PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
+import { faHouse } from '@fortawesome/free-solid-svg-icons';
+
+const modifyUserMenuToggle = ( widget ) => {
+  widget.content.username = "My Account";
```

Comment:
This example makes me wonder if we should rename the `username` prop from something tied to the default content (the username) to something more generic. I don't have any great name ideas (first things that came to mind were "label" and "text" and I don't love either), but if people agree using a more generic term makes sense I'm sure we can think of something.